rudder-core/src/main/scala/com/normation/rudder/services/policies/InterpolatedValueCompiler.scala

```diff
@@ -253,31 +310,29 @@ class InterpolatedValueCompilerImpl extends RegexParsers with InterpolatedValueC
   //exactly quote s - no regex authorized in
   def id(s: String) = ("""(?iu)\Q""" + s + """\E""").r
 
-  def all: Parser[List[Token]] = (rep1(interpol | plainString) | emptyVar)
+  def all: Parser[List[Token]] = (rep1(interpol | nodeProp | plainString) | emptyVar)
 
   //empty string is a special case that must be look appart from plain string.
   //also parse full blank string, because no need to look for more case for them.
-  def emptyVar: Parser[List[CharSeq]] = """(?iums)(\s)*""".r ^^ { x => List(CharSeq(x)) }
+  def emptyVar   : Parser[List[CharSeq]] = """(?iums)(\s)*""".r ^^ { x => List(CharSeq(x)) }
 
-  def plainString: Parser[CharSeq] = ("""(?iums)((?!\Q${rudder.\E).)+""").r  ^^ { CharSeq(_) }
+  // plain string must not match our identifier, ${rudder.* and ${node.properties.*}
+  def plainString: Parser[CharSeq] = ("""(?iums)((?!(\Q${rudder.\E|\Q${node.properties\E)).)+""").r  ^^ { CharSeq(_) }
 
   //identifier for step in the path or param names
-  def propId: Parser[String] = """[\-_a-zA-Z0-9]+""".r
+  def propId     : Parser[String] = """[\-_a-zA-Z0-9]+""".r
```

Comment:
Sorry, I just completly mis transcript / understood. 
All reperition operators are greedy. We don't want there lazy counterparts, for eg ".?", "+?" etc because, yes, in that case we will get partial string. We don't risk getting too much string with greedy operators because we have clear delimiters out of greedy scope. 
"?m" is for multiline. We don't want line break in the middle of word, so we don't want that either (but "space" is matching any whitespace characters, including line breaks, so we get them where space are put (i.e around chars used for separators, like ".", "]", etc))